### ISSUE TITLE
Maps - Banner appears at side on mobile

### ISSUE THREAD

**Issue by DNA-styx** [DNA-styx/HLStatsX-Windmill-Theme#30]
https://stats.dnagames.site/hlstats.php?mode=mapinfo&map=dod_gan_xmas_b1&game=dods

**Comment by stive101** [comment]
hi, im not sure what u mean. 
The hlstatsx Logo at right side ? if u mean that, that should be the map picture if u upload one for the map into the games folder.

**Comment by DNA-styx** [comment]
Yes of course. Thanks. What threw me was in mobile view the banner is off to the side.
